moveit_ros/moveit_servo/include/moveit_servo/make_shared_from_pool.h

```diff
@@ -34,7 +34,7 @@
  * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
  * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-*******************************************************************************/
+ *******************************************************************************/
```

Comment:
This is a generic utility and would much better live in this folder for all of moveit to use:

https://github.com/ros-planning/moveit/tree/master/moveit_core/macros/include/moveit/macros